docs/RESOURCES.md

```diff
@@ -0,0 +1,18 @@
+# Recommended learning resources
+
```

Comment:
Actually it has an h1, but that's just the language name. Let's make this an h2.